pwa/app/routes/event.$eventKey.tsx

```diff
@@ -416,3 +435,33 @@ function TeamsTab({
     </div>
   );
 }
+
+function getTeamRobotPic(media: Media[]): string | undefined {
+  const pics = media.filter((m) =>
+    ['imgur', 'cdphotothread', 'instagram-image', 'external-link'].includes(
+      m.type,
+    ),
+  );
+
+  if (pics.length === 0) {
+    return undefined;
+  }
+
+  const maybePreferred = pics.find((m) => m.preferred);
+
+  function imageUrl(media: Media) {
+    if (media.type === 'imgur') {
+      return `https://i.imgur.com/${media.foreign_key}s.jpg`;
+    }
+
+    if (media.type === 'instagram-image') {
+      return `https://www.thebluealliance.com/instagram_oembed/${media.foreign_key}`;
+    }
+
+    return media.direct_url;
+  }
+
+  return maybePreferred === undefined
+    ? imageUrl(pics[0])
```

Comment:
I don't see a lot of value doing pics[0] here. I think if there aren't any preferred then we just return undefined.